extensions/amp-accordion/1.0/accordion.js

```diff
@@ -189,6 +199,7 @@ export function AccordionSection({
 
   const expanded = isExpanded ? isExpanded(id, defaultExpanded) : expandedState;
   const animate = contextAnimate ?? defaultAnimate;
+  const contentId = `${prefix || 'a'}_AMP_content_${id}-${suffix}`;
```

Comment:
No need for "AMP" here and let's turn underscores to dashes and use lower case.

Reply:
Sounds good updated!